README.md

```diff
@@ -19,18 +19,19 @@
 
 ## Installing
 
-This package requires Julia v0.6 or later. Refer to the [official documentation](https://julialang.org/downloads)
-on how to install and run Julia in your system.
+This package requires Julia v1.0 or later.
+Refer to the [official documentation](https://julialang.org/downloads) on how to
+install and run Julia in your system.
 
-To install the latest release of this package, use the following command inside Julia's REPL:
+To install the latest release of this package, use the following command from
+Julia's REPL (to activate the `pkg` mode, type `]`):
 
 ```julia
-using Pkg
-Pkg.add("LazySets")
+pkg> add LazySets
 ```
 
-If you want to install the latest development version, do:
+To install the latest development version, instead use:
 
 ```julia
-Pkg.clone("https://github.com/JuliaReach/LazySets.jl.git")
+pkg> dev LazySets
```

Comment:
```suggestion
pkg> add LazySets#master
```

Reply:
What is the difference?

Reply:
We can add a link to https://docs.julialang.org/en/v1/stdlib/Pkg/index.html for the different uses (and `dev` in particular).

Reply:
I would say that `dev` is the preferred way to develop a package (because it gets moved to your `dev` folder).
I can add a link to [adding registered packages](https://julialang.github.io/Pkg.jl/v1/managing-packages/#Adding-registered-packages-1) and [developing packages](https://julialang.github.io/Pkg.jl/v1/managing-packages/#Developing-packages-1).
But generally I prefer to keep the instructions short and simple. If one does not know how to use Julia, one has to consult a tutorial.

Reply:
Indeed, `dev` clones the full git repo for development purpose (i don't think this is the case when using `add LazySets#master`).

The updated version looks good to me 👍